NextCenturyCorporation/itm-evaluation-server
# Itm 86: Add gradle config for swagger generation

Same branch as swagger but I renamed it to match the pattern used for other branches.

To test:
Manually delete the swagger_server/models folder
refresh your IDE or do a "git status" on the command line to see that git see the files as missing

run "./gradlew" to run the swagger generation

refresh your IDE or do a "git status" again. If all the models are generated correctly there should be no changes from gits perspective. The only file that is different in the init.py but its only unnecessary comments that are changed.
